src/osd/PrimaryLogPG.cc

```diff
@@ -5700,7 +5707,9 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 	    oi.truncate_size = op.extent.truncate_size;
 	  }
 	}
-	result = check_offset_and_length(op.extent.offset, op.extent.length, cct->_conf->osd_max_object_size);
+        auto dpp = get_dpp();
+	result = check_offset_and_length(op.extent.offset, op.extent.length,
+          osd_max_object_size, dpp);
```

Comment:
@shinobu-x  instead of having a `dpp` local variable, i'd recommend just use `get_dpp()` directly.